src/createAnimatedComponent.js

```diff
@@ -250,17 +250,18 @@ export default function createAnimatedComponent(Component) {
         viewTag = findNodeHandle(this);
         viewName = null;
       } else {
+        // hostInstance can be null for a component that doesn't render anything (render function returns null). Example: svg Stop: https://github.com/react-native-svg/react-native-svg/blob/develop/src/elements/Stop.tsx
         const hostInstance = RNRenderer.findHostInstance_DEPRECATED(this);
         // we can access view tag in the same way it's accessed here https://github.com/facebook/react/blob/e3f4eb7272d4ca0ee49f27577156b57eeb07cf73/packages/react-native-renderer/src/ReactFabric.js#L146
-        viewTag = hostInstance._nativeTag;
+        viewTag = hostInstance?._nativeTag;
         /**
          * RN uses viewConfig for components for storing different properties of the component(example: https://github.com/facebook/react-native/blob/master/Libraries/Components/ScrollView/ScrollViewViewConfig.js#L16).
          * The name we're looking for is in the field named uiViewClassName.
          */
-        viewName = hostInstance.viewConfig?.uiViewClassName;
+        viewName = hostInstance?.viewConfig?.uiViewClassName;
         // update UI props whitelist for this view
         if (this._hasReanimated2Props(styles)) {
-          adaptViewConfig(hostInstance.viewConfig);
+          adaptViewConfig(hostInstance?.viewConfig);
```

Comment:
[This will throw](https://github.com/software-mansion/react-native-reanimated/blob/6990128ef50399004692dce589a88df65bd76418/src/ConfigHelper.js#L134-L135) if you pass an `undefined` here.

Reply:
Yes I know but I don't see an option for it to be undefined

Reply:
Then why we have `?.` here?

Reply:
this executes only if you attach UAS result to it but maybe there could be a check here to make it safer just in case

Reply:
hostInstance can be null but not undefined, but the whole expression might be undefined, I misunderstood, sorry

Reply:
fixed 9492c29